plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/ExasolQueryRunner.java

```diff
@@ -39,6 +40,12 @@ public final class ExasolQueryRunner
 {
     private static final Logger log = Logger.get(ExasolQueryRunner.class);
 
+    static {
+        Logging logging = Logging.initialize();
+        logging.setLevel("io.trino.plugin.exasol", Level.TRACE);
```

Comment:
Thats gonna get super noisy. I think this might be fine for a temporary look .. but not permanently. Do we have any better solution or typical approach @martint .. also cc @ebyhr

Reply:
No, this is the right approach to increase logging level temporarily for debugging something in CI.

We should mark the PR in some way so it doesn't get inadvertently merged.